inc/block-patterns.php

```diff
@@ -45,4 +45,14 @@
 			'content'       => '<!-- wp:media-text {"mediaId":1752,"mediaLink":"' . esc_url( get_template_directory_uri() ) . '/assets/images/the_poplars_at_saint_remy.jpg","mediaType":"image","className":"is-style-twentytwentyone-border"} --><div class="wp-block-media-text alignwide is-stacked-on-mobile is-style-twentytwentyone-border"><figure class="wp-block-media-text__media"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/the_poplars_at_saint_remy.jpg" alt="' . esc_attr__( '&quot;The Poplars at Saint-Rémy&quot; by Vincent Van Gogh (1889)', 'twentytwentyone' ) . '" class="wp-image-1752"/></figure><div class="wp-block-media-text__content"><!-- wp:heading {"align":"center"} --><h2 class="has-text-align-center">' . wp_kses_post( __( 'The Poplars at<br>Saint-Rémy', 'twentytwentyone' ) ) . '</h2><!-- /wp:heading --><!-- wp:separator --><hr class="wp-block-separator"/><!-- /wp:separator --><!-- wp:paragraph {"align":"center","fontSize":"small"} --><p class="has-text-align-center has-small-font-size">' . wp_kses_post( __( 'Vincent van Gogh<br>(Dutch, 1853-1890)', 'twentytwentyone' ) ) . '</p><!-- /wp:paragraph --></div></div><!-- /wp:media-text -->',
 		)
 	);
+
+	register_block_pattern(
+		'twentytwentyone/link-cta',
+		array(
+			'title'         => __( 'Link CTA', 'twentytwentyone' ),
```

Comment:
What is a CTA? I understand that you mean call to action, but can this title be improved?
Or a description added.

Reply:
Yeah, I just grabbed this from the issue. @melchoyce do you have a different suggestion?

Reply:
In https://github.com/WordPress/twentytwentyone/pull/174/commits/844eb51486c174fb73f6a83902cee763bf803c7a, I renamed it to "Links Area" for now.